### DISCUSSION TITLE
Controller class not found

### DISCUSSION THREAD

**Discussion by Logos456** [livewire/livewire#4071] [1. Help]
in Laravel web.php I have the following code:
1. use App\Http\Livewire\Users\ProgramIndex;
2. Route::get('/programs/{program:slug}', ProgramIndex::class)->name('programIndex');

Works locally on WAMP, but when I deploy to Forge, I get the following error when I click DEPLOY NOW. Not sure why...Any help would be greatly appreciated. Thanks

App\Http\Livewire\Users\ProgramIndex` was not found: Controller class

**Comment by joshhanley** [comment]
@Logos456 that's a tough one, and not something I think we can really help troubleshoot as you will need to go through your server yourself and try to work out what is happening.

The best thing I think you can do would be to log into the server via SSH and go to your site's directory and make sure that the class has been loaded correctly.

Then maybe run `composer dump-autoload` on your server to make sure it's picking it up correctly.

Hope this helps!

**Reply by Logos456** [reply]
Thanks for the help! I SSH'd into the server, checked that the files were properly loaded, and ran composer dump-autoload. Same error, unfortunately, but thank for the suggestions!

**Reply by joshhanley** [reply]
@Logos456 bugger! That is really strange. Hope you can work it out!
